### PR TITLE
Fix chrome enabling multiple themes at once

### PR DESCRIPTION
**Pull request by BlankParenthesis**
Since Chrome does not fire the load event for elements that have already loaded being added back to the DOM, the code that cleaned up previous themes would not fire when switching back to a theme that had previously been loaded.

This fix changes the way themes are loaded such that they stay on the page and are disabled with the `disabled` property until needed again. In addition, a `loaded` variable is also stored to determine if the app needs to add the newly selected theme to the DOM.

### CODE REVIEW

File: resources/public/pxls.js [hunk 1]
```diff
@@ -3361,6 +3346,45 @@ window.App = (function() {
...
+          return;
+        }
+        if (!(index in self.themes)) {
+          return console.warn(`Tried to load invalid theme "${index}"`);
+        }
+        const theme = self.themes[index];
+        if (theme.loaded) {
+          self.enableTheme(index);
+        } else {
+          await new Promise((resolve, reject) => {
```
Comment (netux) [inline resources/public/pxls.js:3363]: Why does this need to be wrapped inside a Promise?

Comment (BlankParenthesis) [reply resources/public/pxls.js:3363]: It doesn't strictly have to be, but this way functions calling `loadTheme`  can await until the theme actually gets loaded.

Unless `JQuery.one` returns a Promise which don't see any documentation of.

Comment (netux) [reply resources/public/pxls.js:3363]: If some code ever needs to wait for a theme to load, then this can be readded; but right now it has no use.

Comment (BlankParenthesis) [reply resources/public/pxls.js:3363]: Can I get a second opinion on this?
Waiting for the theme to load before returning seems like the *correct* behaviour for this function regardless of whether any code makes use of that feature.

Comment (FlyingSixtySix) [reply resources/public/pxls.js:3363]: I think it'd be good to have anyway for the reason @BlankParenthesis noted.

**Pull request merged**